ninja/patch_dict.py

```diff
@@ -22,10 +30,32 @@ def _validate(cls, input_value: Any) -> Any:
         return input_value.model_dump(**cls._wrapped_model_dump_params)
 
 
+def get_schema_annotations(schema_cls: Type[Any]) -> Dict[str, Any]:
+    schema = schema_cls
+    schemas = []
+    excluded_bases = (
+        "<class 'ninja.schema.Schema'>",
+        "<class 'ninja.orm.ModelSchema'>",
+        "<class 'pydantic.main.BaseModel'>",
```

Comment:
I think you can use actual classes here instead of their repr (which may break in some potential new version)